discover.py

```diff
@@ -0,0 +1,59 @@
+from itertools import chain
+import requests
+from typing import Dict, NamedTuple, List
+
+
+WorkerName = str
+TaskName = str
+TaskType = str
+Task = NamedTuple('Task', [
+    ('type', TaskType),
+    ('name', TaskName),
+])
+
+REQUEST_URL = "celery-flower:2004/api/workers?refresh=True"
+
+
+def workers() -> Dict[WorkerName, List[Task]]:
+    """Find workers available in the cluster
+
+    Returns:
+        Dictionary of workers with their tasks.
+    """
+    workers_info = requests.get(REQUEST_URL).json()
```

Comment:
Forgot this one; we really need to get available task names directly from Celery/RabbitMQ instead of flower, which is just an implementation detail at this point. This should be refactored in a separate merge request.

Reply:
[`celery.task.control`](https://stackoverflow.com/a/26211200) seems like a viable solution to this problem.

Reply:
Accepted.